resources/public/api/conf/2.0/scenarios/retrieveBusinessIncome.md

```diff
@@ -0,0 +1,23 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+<table>
+    <thead>
+        <tr>
+            <th>Header Value (Gov-Test-Scenario)</th>
+            <th>Scenario</th>
+        </tr>
+    </thead>
+    <tbody>
+            <tr>
+                <td><p>N/A - DEFAULT</p></td>
+                <td><p>Simulates success response with profit.</p></td>
+            </tr>
+            <tr>
+                <td><p>NOT_FOUND</p></td>
+                <td><p>Simulates the scenario where no data is found.</p></td>
+            </tr>
+            <tr>
+                <td><p>LOSS</p></td>
+                <td><p>Simulates success response with loss.</p></td>
+            </tr>
+        </tbody>
```

Comment:
I remember there being a discussion in the stand up that there should be a scenario for a property response and a self-employment response as well as the profit & loss ones